.sqlfluff

```diff
@@ -33,13 +25,12 @@ exclude_rules = aliasing.column,
     references.from,
     references.keywords,
     references.qualification,
-    references.quoting,
     structure.column_order,
     structure.distinct,
+    # While explicit, this provides clarity.
     structure.else_null,
```

Comment:
This rule bans `ELSE NULL`. I don't see a good reason for doing so, it helps clarity IMO. https://github.com/sqlfluff/sqlfluff/issues/536